pkg/sypgp/sypgp.go

```diff
@@ -103,7 +103,11 @@ func GetTokenFile() string {
 
 // DirPath returns a string describing the path to the sypgp home folder
 func DirPath() string {
-	return filepath.Join(syfs.ConfigDir(), "sypgp")
+	sypgpDir := os.Getenv("SINGULARITY_SYPGPDIR")
```

Comment:
Probably not ideal to ask for the variable here, but its done like this in other files...